playground/src/examples/twitch-reset-password.vue

```diff
@@ -1,31 +1,108 @@
 <template>
-  <e-html>
-    <e-head />
-    <e-preview>You updated the password for yout Twitch account</e-preview>
-    <e-tailwind>
-      <e-body class="bg-[#efeef1] font-['HelveticaNeue,Helvetica,Arial,sans-serif']">
+  <e-tailwind>
```

Comment:
I'm not sure if using Tailwind as the root component would be convenient, as `e-head` and `e-preview` don't require styles. Tell me whats is your point?